buildSrc/src/main/kotlin/buildsrc/conventions/lang/kotlin-multiplatform-js.gradle.kts

```diff
@@ -0,0 +1,16 @@
+package buildsrc.conventions.lang
+
+/** conventions for a Kotlin/JS subproject */
+
+plugins {
+    id("buildsrc.conventions.lang.kotlin-multiplatform-base")
+}
+
+kotlin {
+    targets {
+        js(IR) {
+            browser()
+            nodejs()
```

Comment:
I also enabled a NodeJS target. I don't think this has any effect on the published artifacts, but it will mean that NodeJS is tested.